ci-operator/step-registry/cucushift/sdn-live-migration-ovn/cucushift-sdn-live-migration-ovn-chain.yaml

```diff
@@ -3,9 +3,12 @@ chain:
   steps:
   - ref: idp-htpasswd
   - ref: cucushift-pre
+  - ref: openshift-extended-upgrade-pre  
   - ref: cucushift-upgrade-prepare
+  - ref: cucushift-upgrade-prehealthcheck
   - ref: cucushift-sdn-live-migration-ovn
+  - ref: openshift-extended-upgrade-post
   - ref: cucushift-upgrade-check
   - ref: openshift-upgrade-qe-test-report
   documentation: |-
-    The workflow executes cucushift sdn live migration to ovn
\ No newline at end of file
+    The workflow executes cucushift sdn live migration to ovn
```

Comment:
Shall we update the doc to reflect the change, too ?

Reply:
Yes,  updated, thanks